sw/device/tests/BUILD

```diff
@@ -1691,6 +1691,20 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
+    name = "rv_timer_systick_test",
+    srcs = ["rv_timer_systick_test.c"],
+    deps = [
+        "//hw/top_earlgrey/sw/autogen:top_earlgrey",
+        "//sw/device/lib/dif:rv_timer",
+        "//sw/device/lib/runtime:hart",
+        "//sw/device/lib/runtime:ibex",
+        "//sw/device/lib/runtime:irq",
+        "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing/test_framework:ottf_main",
+    ],
+)
```

Comment:
Use `opentitan_test` which will be rule supporting sival targets.

```python
opentitan_test(
    name = "rv_timer_systick_test",
    srcs = ["rv_timer_systick_test.c"],
    exec_env = EARLGREY_TEST_ENVS,
    deps = [
        "//hw/top_earlgrey/sw/autogen:top_earlgrey",
        "//sw/device/lib/dif:rv_timer",
        "//sw/device/lib/runtime:hart",
        "//sw/device/lib/runtime:ibex",
        "//sw/device/lib/runtime:irq",
        "//sw/device/lib/runtime:log",
        "//sw/device/lib/testing/test_framework:ottf_main",
    ],
)
```

Reply:
Done